hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/downloader/AbstractDownloaderTest.java

```
@@ -285,15 +290,10 @@ private void startS3Proxy() throws Exception {
                 .build();
         s3Proxy.start();
 
-        for (int i = 0; i < 500; i++) {
-            if (s3Proxy.getState().equals(AbstractLifeCycle.STARTED)) {
-                return;
-            }
-
-            Thread.sleep(1);
-        }
-
-        throw new RuntimeException("Timeout starting S3Proxy, state " + s3Proxy.getState());
+        await("S3Proxy").dontCatchUncaughtExceptions()
+                .atMost(Duration.ofMillis(500))
+                .pollDelay(Duration.ofMillis(1))
+                .until(() -> AbstractLifeCycle.STARTED.equals(s3Proxy.getState()));
```

Do we still need to throw an exception at the end of the polling?